packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -87,6 +87,12 @@
             "valueString":"{{{parseReferenceData observationEntry.value._}}}",
         {{!-- in some cases text is here in the b64 field. Adding so no data loss but not sure if thats the way its supposed to be. also how it renders I do not know      --}}
         {{else if observationEntry.value._b64}}
+            "extension": [
+                {
+                    "url": "https://public.metriport.com/fhir/StructureDefinition/is-base-64-encoded.json",
+                    "valueBoolean": true
+                }
+            ],
```

Comment:
we could introduce a dependency on core in the fhir-converter, and then have the extension here come from handlebars-helpers.js with a function that imports from base64-extension.ts

We dont want to change one in one place and forgot to change the other. Con here would be I think currently the converter has no dependencies on the rest of the product so far. 

Wdyt?

Reply:
Yea, I was hesitant to create this dependency, but I also realize the consequences of not importing it here.. Not sure what the best approach is. Maybe add TSDoc comments in both places? 🤷

Reply:
Yah tsDoc in both is good for me